OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,877 +1,1523 @@
+        {
+            ImGui::SameLine();
+            ImGui::TextColored(DebugUtils::Colors::Warning, "Capturing next frame...");
         }
     }
-    void CommandPacketDebugger::RenderPerformanceStats()
+
+    // ========================================================================
+    // Frame Selector
+    // ========================================================================
+
+    void CommandPacketDebugger::RenderFrameSelector()
```

Clang (14.0.6)

[warning] method 'RenderFrameSelector' can be made static

(readability-convert-member-functions-to-static)